src/Okta.Sdk.IntegrationTests/GroupScenarios.cs

```diff
@@ -38,10 +39,30 @@ public class GroupScenarios : ScenarioGroup
                 () => client.Groups.GetGroupAsync(createdGroup.Id));
         }
 
-        [Fact(Skip = "TODO")]
+        [Fact]
         public async Task ListGroups()
         {
-            throw new NotImplementedException();
+            var client = GetClient("group-list");
+
+            var createdGroup = await client.Groups.CreateGroupAsync(new CreateGroupOptions
+            {
+                Name = "List Test Group",
+            });
+
+            try
+            {
+                var groupList = await client.Groups.ListGroups().ToArray();
+                groupList.SingleOrDefault(g => g.Id == createdGroup.Id).Should().NotBeNull();
+            }
+            finally
+            {
+                // TODO: add this to IGroup
+                await client.Groups.DeleteGroupAsync(createdGroup.Id);
```

Comment:
I did in fact add it to IGroup 😄 can you update this to use `createdGroup.DeleteAsync`?